tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
     "compilerOptions": {
-        "module": "es2015",
+        "module": "commonjs",
```

Comment:
I think it should also work without this change

Reply:
When importing the package I get strange unexpected import token which makes it impossible to use.